modules/candidate_list/jsx/onLoad.js

```diff
@@ -0,0 +1,62 @@
+/* global formatColumn */
```

Comment:
We are usually naming this file `something...Index` or just `index`

Reply:
I'll rename the component to be CandidateListIndex and separate it into 2 files: one for the react component, and one for the onLoad handler..